src/errors/runtime/templates.js

```diff
@@ -107,4 +107,5 @@ export default {
         'If you use a portable browser version, ' +
         'specify the browser alias before the path instead of the \'path\' prefix. ' +
         'For more information, see https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#test-in-headless-mode',
+    [RUNTIME_ERRORS.uncaughtErrorInReporter]: 'An uncaught error occured in a reporter:\n{error}',
```

Comment:
I think it won't be hard to include the reporter method name in the message. If the reporter method is complex, the method name won't be at the top of the stack, and it can be hard to find it.

Reply:
Just a note, we usually add an empty line after multi-line expression